src/task2/db/users.js

```diff
@@ -0,0 +1,24 @@
+import uuid from 'uuid/v1';
+
+import UserEntity from '../entitites/User';
+import { deleteItemById, getItemsByPageAndLimit } from '../utils/commonUtils';
+
+const users = [];
+
+export const getUsers = (page, limit) => getItemsByPageAndLimit(users, page, limit);
+export const getUserById = (id) => users.find(user => user.id === id);
+export const addUser = (data) => {
+    const id = uuid();
+    const userData = { ...data, id };
+    users.push(new UserEntity(userData));
+    return getUserById(id);
+};
+export const updateUser = (id, data) => {
+    const index = users.findIndex(user => user.id === id);
+    if (index >= 0) {
+        users[index] = data;
+        return users[index];
+    }
+    throw new Error('Cannot find');
```

Comment:
Consider storing such strings in the core folder and reuse them across the project.